cg/apps/coverage/api.py

```diff
@@ -87,3 +88,57 @@ def sample_coverage(self, sample_id: str, panel_genes: list) -> dict:
         return ReadStream.get_content_from_stream(
             file_format=FileFormat.JSON, stream=self.process.stdout
         ).get(sample_id)
+
+
+class Chanjo2API:
+    """Interface to Chanjo2, the coverage analysis tool"""
+
+    def __init__(self, base_url, config):
+        self.base_url = base_url
+        self.chanjo2_config = config["chanjo2"]["config_path"]
+        self.chanjo2_binary = config["chanjo2"]["binary_path"]
+        self.process = Process(binary=self.chanjo2_binary, config=self.chanjo2_config)
+
+    def get_coverage_statistics(
+        self, coverage_file_path, intervals_bed_path, completeness_thresholds
+    ):
+        endpoint = f"{self.base_url}/coverage/d4/interval_file/"
```

Comment:
This looks OK I think. Should be called for each sample of the case and it returns a response like this:

<img width="409" alt="image" src="https://github.com/Clinical-Genomics/cg/assets/28093618/c2bb4234-dff3-4488-850b-605733068883">

What is the exact data that you need to include in the delivery report?

Reply:
thanks! It should be median coverage over the omim panel, and the completeness at 10X (medelsekvensdjup & täckningsgrad in the delivery report).
<img width="901" alt="image" src="https://github.com/Clinical-Genomics/cg/assets/57712924/2c26d79c-d31c-46b0-916d-5946968fd7d6">